lib/generated/breez_translations.dart

```diff
@@ -95,8 +95,8 @@ abstract class BreezTranslations {
 
   /// A list of this localizations delegate's supported locales.
   static const List<Locale> supportedLocales = <Locale>[
-    Locale('en'),
     Locale('de'),
+    Locale('en'),
```

Comment:
Could you revert this order as it causes:
- #1

More explanation here: https://github.com/breez/Breez-Translations/commit/9c8cc35d9e289a9696ab15ad91734ff12739b171

Reply:
Rebased to main, does it look correct now ?

Reply:
No, rebasing won't fix this as you're still adding this change on top of main. You need to manually revert the changes with another commit.